### PR TITLE
Fix install language

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.x
| Description?  | Fix install language
| Type?         | bug fix
| Category?     | IN
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | check admin.php, errors.php, etc file on translations after installing

### CODE REVIEW

**Comment by vincentbz** [comment]
futurama

**Comment by aleeks** [comment]
Dirty fix for the moment because it's working during the modules update.. and not because we want :smile: 

ping @kpodemski

**Comment by vincentbz** [comment]
Too dirty to be tested ? 😇

**Comment by aleeks** [comment]
Yep, i will update this morning!

**Comment by kpodemski** [comment]
Yes, it's working @vincentbz @aleeks 

Good job 👍

**Comment by vincentbz** [comment]
Hell yeah !

**Review by maximebiloe — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you @aleeks & @kpodemski for the help

File: classes/Language.php [hunk 1]
```diff
@@ -960,7 +960,17 @@ public static function downloadAndInstallLanguagePack($iso, $version = null, $pa
                 $files_list = $other_files;
             }
 
-            if (!$gz->extractList(AdminTranslationsController::filesListToPaths($files_list), _PS_TRANSLATIONS_DIR_.'../')) {
+            // don't know why, but it's needed & doesn't work otherwise
```
Comment (hibatallahAouadni) [inline classes/Language.php]: @aleeks the translations_extract is needed and I believe it works, so which part it doesn't work?

Comment (aleeks) [inline classes/Language.php]: Maybe in some case, it doesn't work.. I don't know why we did it like that.

**Pull request merged**